src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -639,7 +635,12 @@ std::unique_ptr<data_iterator> activity_classifier::create_iterator(
   data_params.prediction_window = read_state<flex_int>("prediction_window");
   data_params.predictions_in_chunk = NUM_PREDICTIONS_PER_CHUNK;
   data_params.use_data_augmentation = use_data_augmentation;
-  data_params.random_seed = read_state<int>("random_seed");
+  if (!use_data_augmentation){
+    data_params.random_seed = 0;
+  }
+  else{
+    data_params.random_seed = read_state<int>("random_seed");
+  }
```

Comment:
Did you run clang-format here? In most of our code, the formatting should be
```
if (condition) {
  ...
} else {
  ...
}
```

Reply:
hmmm it seems like the clang-format didn't run when I commit it.
Will take a look